packages/models/src/lib/global-cli-options.ts

```diff
@@ -0,0 +1,44 @@
+import { z } from 'zod';
+
+/**
+ * Define Zod schema for the GlobalCliArgs type
+ *
+ * @example
+ *
+ * // Example data for the GlobalCliArgs type
+ * const uploadConfigData = {
+ *   server: 'https://example.com/api',
+ *   apiKey: 'your-api-key',
+ * };
+ *
+ * // Validate the data against the schema
+ * const validationResult = globalCliArgsSchema.safeParse(globalCliArgsData);
+ *
+ * if (validationResult.success) {
+ *   console.log('Valid config:', validationResult.data);
+ * } else {
+ *   console.error('Invalid config:', validationResult.error);
+ * }
+ *
+ */
+export const globalCliArgsSchema = z.object({
+  interactive: z
+    .boolean({
+      description:
+        'flag if interactivity should be considered. Useful for CI runs.',
+    })
+    .default(true),
+  verbose: z
+    .boolean({
+      description: 'Outputs additional information for a run',
+    })
+    .default(false),
+  configPath: z
+    .string({
+      description: "Path to config.js. defaults to 'cpu-config.js'",
+    })
+    .optional()
+    .default('cpu-config.js'),
```

Comment:
Let's stick to one working title for now, before we settle on a product name. CPU is a very misleading acronym anyway.

```suggestion
  configPath: z
    .string({
      description: "Path to config.js. defaults to 'qm-config.js'",
    })
    .optional()
    .default('qm-config.js'),
```

or 

```suggestion
  configPath: z
    .string({
      description: "Path to config.js. defaults to 'quality-metrics.js'",
    })
    .optional()
    .default('quality-metrics.js'),
```

Reply:
fixed